dfdaemon/config/config.go

```diff
@@ -130,13 +131,12 @@ func (p *Properties) Validate() error {
 		)
 	}
 
-	if _, err := os.Stat(p.DFPath); err != nil && os.IsNotExist(err) {
-		return dferr.Newf(
-			constant.CodeExitDfgetNotFound,
-			"dfpath %s not found", p.DFPath,
-		)
-	}
-
+	// if _, err := os.Stat(p.DFPath); err != nil && os.IsNotExist(err) {
```

Comment:
just remove them if you think the code is useless

Reply:
done.